app/view_models/profile_index.rb

```diff
@@ -62,10 +62,10 @@ def manage_personal_key_partial
   def header_personalization
     return decrypted_pii.first_name if decrypted_pii.present?
 
-    current_user.email
+    decorated_user.email
```

Comment:
Alright, swapped `SimpleDelegator` for explicit delegation at: 9b093f0c -- what do y'all think?